symfony/plugins/orangehrmRESTPlugin/lib/Api/Leave/LeaveRequestAPI.php

```diff
@@ -372,4 +525,31 @@ protected function createParameters($filters)
         return new \ParameterObject($parameters);
     }
 
+    /**
+     * Return accessible leave list employees for current request user
+     * @param bool $withTerminated
+     * @return array
+     * @throws \ServiceException
+     */
+    protected function getAccessibleEmployeeIds($withTerminated)
+    {
+        $properties = array("empNumber", "firstName", "middleName", "lastName", "termination_id");
+        $requiredPermissions = ['action' => ['view_leave_list']];
+
+        $employeeList = \UserRoleManagerFactory::getUserRoleManager()->getAccessibleEntityProperties('Employee',
```

Comment:
better to follow PSR4 coding standard for multifunctional lines

Reply:
Done